Mayhemfile

```diff
@@ -0,0 +1,16 @@
+# Namespaced project name that the target belongs to
+project: bee
+
+# Target name (should be unique within the project)
+target: bee-message
+
+# Base image to run the binary in.
+image: ghcr.io/prateekvishnu/bee-message:latest
```

Comment:
For this (and all other repositories), there is no need to specify the base image as the image is provided, additionally this will changed based on the repository. Please remove or comment out the image field.